src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <ProjectGuid>{F24D3391-2928-4E83-AADE-B34423498750}</ProjectGuid>
     <AssemblyName>System.Threading.Tasks.Extensions</AssemblyName>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
I've been told that adding APIs requires bumping the minor version.  @weshaggard, is that correct?